packages/lambda-powertools-middleware-correlation-ids/index.js

```diff
@@ -30,3 +29,5 @@ module.exports = ({ sampleDebugLogRate }) => {
     }
   }
 }
+
+module.exports.Log = require('@dazn/lambda-powertools-logger')
```

Comment:
Personally I'd rather we didn't do this - I know the packages are currently coupled (middleware-correlation-ids requires logger), but this feels like it'll introduce more debt and make it harder to move things in the future.\

Additionally, [`const { Log } = require('@dazn/lambda-powertools-middleware-correlation-ids')`](https://github.com/getndazn/dazn-lambda-powertools/pull/220/files#diff-596efdbaae8b2f76f0ddc28a63045e7fR4) doesn't feel right, it raises questions such as "is the middleware-correlation-ids logger different to the normal logger?"

Related ticket to allow using custom logger - https://github.com/getndazn/dazn-lambda-powertools/issues/98, which would also add to the potential confusion

Reply:
Yea this makes a lot of sense!